tool/tsh/proxy.go

```diff
@@ -212,6 +213,8 @@ func sshProxy(tc *libclient.TeleportClient, params sshProxyParams) error {
 
 	args := []string{
 		"-A",
+		// TODO: remove once we finish deprecating SHA1
+		"-o", fmt.Sprintf("PubkeyAcceptedKeyTypes=+%s", ssh.CertAlgoRSAv01),
```

Comment:
Is it `PubkeyAcceptedKeyTypes` or `PubkeyAcceptedAlgorithms` ?

I tried to test this, but no luck
Do we have a guide I can follow to set up the `tsh proxy ssh` ?

Reply:
In https://github.com/gravitational/teleport/pull/11178#issuecomment-1069658244:

> Teleport versions after https://github.com/gravitational/teleport/pull/9205 need only PubkeyAcceptedKeyTypes to interop, as that PR "fixes" SSH signature/kex (our certificates still use [ssh-rsa-cert-v01@openssh.com](mailto:ssh-rsa-cert-v01@openssh.com), even though signing uses [rsa-sha2-512-cert-v01@openssh.com](mailto:rsa-sha2-512-cert-v01@openssh.com))

Providing just `PubkeyAcceptedKeyTypes` seems to work in my testing.

`tsh proxy ssh` is generally used in the `ProxyCommand` in the SSH config generated by `tsh config`, but you can run it directly with `tsh proxy ssh --proxy <proxy_addr> <login>@<node_name>:<node_port>`.

Reply:
Yeah, I tested this with PubkeyAcceptedKeyTypes also and it seemed to work.